docs/src/appendix/cheatcodes/spy_events.md

```diff
@@ -164,6 +165,8 @@ The flow is much simpler (thanks to `EventAssertions` trait). Let's go through i
 
 - After the assertion, found events are removed from the spy. It stays clean and ready for the next events.
 
+In cases where you want to test an event was *not* emitted, use the `assert_not_emitted` function. It works similarly as `assert_emitted` with the only difference that it fails if an event was emitted during the execution.
```

Comment:
It's always good to add even some very simple code example, so please do it :)
Also, please split this very long line 🙏

Reply:
It would be good to make a paragraph for that case. 
`## Ensuring the event was not emitted` maybe?

Reply:
Done.